package.json

```diff
@@ -52,5 +52,9 @@
     "lerna": "lerna",
     "nuke": "rm -r node_modules; for d in packages/*/node_modules; do echo $d; rm -r $d; done",
     "yalc:all": " for d in packages/*/; do echo $d; cd $d; yalc push --replace; cd '../../'; done"
+  },
+  "dependencies": {
```

Comment:
🔧 
This is not the right place for any package dependencies. This should be moved to the `clerk-js` pacakge.